### DISCUSSION TITLE
Size on disk much larger than Immich reports

### DISCUSSION THREAD

**Discussion by mpfl** [immich-app/immich#5884] [Q&A]
When looking in Immich-> Administration, it reports that it has 88GB of photos stored.

However, in `library`:

```
$ du -sh
152G .
```

I feel like a ~70GB discrepancy is a fair bit to be explained by filesystem overhead.

Anyone know what's going on?

**Comment by bo0tzz** [comment]
The report in the admin panel indicates the total size of the original files. It does not count thumbnails, encoded videos, etc.

**Reply by mpfl** [reply]
I understand that, but `thumbs` and `encoded-video` were not included in the `library` total.

```
24G  encoded-video/
152G library/
16G  thumbs/
```

**Reply by bo0tzz** [reply]
Gotcha, then that sounds like there are orphaned files in there. The server stats report is based on the information in the database, and is thus the size that Immich -expects- the library folder to be.

**Reply by mpfl** [reply]
Using `find library -type f | wc -l` indicates many more files than Immich believes there are, so I'm cheekily abusing the storage templates to move the known files to another location so I can delete what's left.

**Reply by mpfl** [reply]
No, that wasn't it. Pretty much every file was moved during the storage template migration.

Could it actually be that we both have iPhones and therefore have lots of live photos? Immich is counting two files as a single asset, but possibly also not adding the size of both files.

**Reply by bo0tzz** [reply]
Seems like that might indeed be the case. I've turned this into an issue: #5885
